### PR TITLE
Jess april13

### PR DESCRIPTION
**Pull request by JessicaRoseBot**
I applied and revised a grid to Set up page Mini Me & added Justina avataaar.

### CODE REVIEW

File: src/app/mini-me/mini-me-setup/mini-me-setup.page.html [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
+
+      <ion-row style="background-color: blue; size: 50%;"><ion-col>
+      <ion-img class="ml-7 mr-7 mb-10" src="./assets/avataaars/justina.svg"></ion-img></ion-row>
+
+  <ion-row style="background-color: aqua; height: 25%;">
+
+    <ion-button><h4 style="color: white;">Let's do it</h4></ion-button></ion-row> -->
+
+<ion-content>
+<ion-grid style="height: 100%;">
```
Comment (kaisenl) [inline src/app/mini-me/mini-me-setup/mini-me-setup.page.html:29]: Generally you want to tab this over by one (two spaces in our case) so you can quickly see where tags start and end.

**Review by kaisenl — COMMENTED** [review]
Can you test if the swipe still works? I had a hard time swiping to the next screen (cc @dginelsa)

Other than that, visually looks good. Definitely looks like @justinajong

File: src/app/mini-me/mini-me-setup/mini-me-setup.page.html (hunk 1 above)
Comment (dginelsa) [reply src/app/mini-me/mini-me-setup/mini-me-setup.page.html:29]: Confirmed that the swipe still works when viewing on the actual iPhone simulator. There's still the blue and teal background that is visible. Normally I would remove it before a pull request. I would approve this as a work in progress? What do you think @kaisenl ?

Comment (kaisenl) [reply src/app/mini-me/mini-me-setup/mini-me-setup.page.html:29]: Cool, sounds good. I can test on a phone after it merges. As for the colors, it's fine for now as a work in progress I think.

**Review by kaisenl — APPROVED** [review]
:shipit:

**Pull request merged**